Allow the newer/older entries paging controls on blogs to be hidden

Kishi85 opened the pull request:
This is useful if you just want to show the top n posts of a blog on a page.

Example:  This is used on www.binary-kitchen.de to just give a top 2 post overview of the kitchen log on the start page without giving the user the option to display older posts as those are available on a separate page.

michitux commented:
Is "pager" really the correct name? I think "paginator" would be better, see e.g. https://jekyllrb.com/docs/pagination/. Apart from that this looks good.

Kishi85 commented:
> Is "pager" really the correct name? I think "paginator" would be better, see e.g. https://jekyllrb.com/docs/pagination/. Apart from that this looks good.

I was assuming the naming scheme applied by Bootstrap for pagination and pager which differences both terms the following way:
- Pagination: Provide pagination links for your site or app with the multi-page pagination component (https://getbootstrap.com/docs/3.3/components/#pagination)
- Pager: Quick previous and next links for simple pagination implementations (or older/newer like in this case, see also https://getbootstrap.com/docs/3.3/components/#pagination-pager)

Following this definition "pager" seems to be the correct choice.

michitux commented on helper.php:
```diff
@@ -163,6 +163,13 @@ function getFlags($setflags) {
+            $flags['pager'] = False;
+        } else {
+            $flags['pager'] = True;
+        }
```
One more minor thing I noticed: It would be nicer to have this as `$flags['pager'] = !in_array('nopager', $setflags);` - or is there any reason to have this if/else statement?

Kishi85 commented:
I've rebased the pull request and changed the flag parsing (the if/else was an oversight on my side and is now corrected). I've also change the flag name to 'nopagingcontrols' to avoid any ambiguity and describe directly what the flag does.
